src/App.css

```diff
@@ -111,7 +119,287 @@ a:hover {
   padding: 0.1rem 0.8rem;
   border-radius: 4px;
 }
-  .open-hours h3 {
-    font-family: "Edu AU VIC WA NT Hand", cursive;
-    font-weight: 300;
-  }
\ No newline at end of file
+
+.open-hours h3 {
+  font-family: "Edu AU VIC WA NT Hand", cursive;
+  font-weight: 300;
+}
+/* Menu Page */
+.menu-page {
+  font-family: "Edu AU VIC WA NT Hand", cursive;
+  padding: 100px;
+  background-color: #000000dd;
+  text-transform: uppercase;
+}
+
+.menu-container {
+  max-width: 900px;
+  margin: 0 auto;
+  background: #fff;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+}
+
+/* Tabs */
+.menu-tabs {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 0px;
+}
+
+.menu-tabs button {
+  background: black;
+  color: white;
+  border: 1px solid #ccc;
+  padding: 10px 20px;
+  font-size: 1rem;
+  text-transform: uppercase;
+  cursor: pointer;
+  transition: 0.3s;
+  flex: 1;
+  font-family: "Edu AU VIC WA NT Hand", cursive;
+}
+
+.menu-tabs button.active {
+  background-color: #e63946;
+  color: white;
+}
+
+.menu-tabs button:not(.active):hover {
+  background-color: #333;
+}
+
+/* Menu Items */
+.menu-title {
+  text-align: center;
```

Comment:
Please create separate css file for Menu component

Reply:
fixed